sql/api/src/main/scala/org/apache/spark/sql/Row.scala

```diff
@@ -219,39 +219,34 @@ trait Row extends Serializable {
    * Returns the value at position i as a primitive boolean.
    *
    * @throws ClassCastException when data type does not match.
-   * @throws NullPointerException when value is null.
```

Comment:
Could you clarify why did you remove it instead of replacing `NullPointerException` by `SparkException`, just wonder.

Reply:
Oops I misunderstood your previous comment. Replaced NPE with `SparkException`.

Reply:
And one more thing is need to update the SQL migration guide because this might impact user apps and potentially break them. Please, add an item to `sql-migration-guide.md`.